whatlies/embeddingset.py

```diff
@@ -580,6 +592,96 @@ def plot_correlation(self, metric=None):
         # Rotate the tick labels and set their alignment.
         plt.setp(ax.get_xticklabels(), rotation=90, ha="right", rotation_mode="anchor")
 
+    def plot_similarity(self, metric="cosine", norm=False):
+        """
+        Make a similarity plot. Shows you the similarity between all the word embeddings in the set.
+
+        Arguments:
+            metric: 'cosine' or 'correlation'
```

Comment:
Oh, sorry! I think I didn't explained it in full. The backticks (`) should remain there (so that they could be shown properly, i.e. as inline code snippet, in the documentation).

Reply:
Ah! Good call.